_includes/hero.html

```diff
@@ -1,13 +1,28 @@
-<section class="container hero is-primary is-bold is-medium">
+<section class="container is-primary is-bold is-medium">
   <div class="hero-body columns is-vcentered">
+    <div class="column">
+      <img src="{{ site.logoNegative | url }}" alt="Servo logo" style="padding: 1rem;" />
```

Comment:
The large logo looks very odd to me. Maybe we can have a screenshot of Servo window here?

Reply:
Did a quick mockup using one of the existing screenshots:
![servo-home-alternate](https://github.com/servo/servo.org/assets/251518/16d206cf-7678-473e-9be8-b0592374ad51)

Reply:
Can you put a screenshot of how it looks to you?

Reply:
This is how it looks for me.

![Screenshot with servo logo](https://github.com/servo/servo.org/assets/11602/1d6fe527-0037-48f7-b8b5-a40d599f09b8)

Reply:
Ok, I got what you meant, that there's a lot of black background there, right. I guess we can try something like you suggest.

Reply:
I have added a screenshot, though I've also kept the big logo as I kind of like it.

![Screenshot of servo.org frontpage, with logo description and screenshot](https://github.com/servo/servo.org/assets/11602/a2183612-3d2c-4426-8917-6e8dedd2bc7b)

How does it look like now?

Reply:
Hmm, it still seems odd and now more 'noisy' as well, compared to [this](https://user-images.githubusercontent.com/251518/267542472-16d206cf-7678-473e-9be8-b0592374ad51.png). The one I linked to feels more clean and polished. However, that might be just me, so maybe we can decide based on input from others as well?

Reply:
IMO smaller logo would be better. Also old screenshot looks better at top edge.

Reply:
Ok, I'll remove the big logo :innocent:. Something like:
![Screenshot of servo.org frontpage without big logo](https://github.com/servo/servo.org/assets/11602/5b25a0c7-c61b-4740-b92e-77dcba374187)

Old screenshot is using legacy layout, I'm not sure we want to show that one. Another option would be to not show a screenshot until it looks better and put there the YouTube video. And have a few more blog posts. For example:
![Screenshot of servo.org frontpage without screenshot and with YouTube video and 4 blog posts](https://github.com/servo/servo.org/assets/11602/a2d60136-3c5c-4065-bac4-0896c8415667)

WDYT?

Reply:
Switching to old wiki layout (vector without 2022) looks good also on new layout engine:
![slika](https://github.com/servo/servo.org/assets/16504129/95b0ad6e-71c2-4fb7-9f5f-8042aea41d6c)

I just do not know how to make screenshot with fake title bar like https://marketplace.visualstudio.com/items?itemName=adpyke.codesnap

Reply:
Somebody that has access to mac can make screenshot for https://en.wikipedia.org/wiki/Servo_(software)?useskin=vector

Reply:
I like this version better:
<img width="795" alt="image" src="https://github.com/servo/servo.org/assets/94557773/ebdb55ed-94c4-42e6-83c7-1c609d3d0c55">

Reply:
Thanks for the feedback, and yeah we can get a better screenshot. I guess we can discuss adding a screenshot in a next PR and so far go with the option that only adds the new project description.